akvo/rsr/spa/build/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Akvo RSR</title>
+    <link href="/my-rsr/styles/style.css" rel="stylesheet">
+</head>
+<body>
+<div id="root"></div>
+<script type="text/javascript" src="/my-rsr/bundle.js"></script>
```

Comment:
As it is, the `bundle.js` cannot be cached. I guess that we will implement some kind of caching, like adding a hash to the js file name?

Reply:
Sure we can add a hash prefix in order to prevent caching. Is that what you meant?

Reply:
Yes. For the MVP, not required, but required for the production launch.

Reply:
Actually this file is targeted by webpack-dev-server in development mode only, for production webpack use `akvo/rsr/spa/app/index.html` and inject the css & js tag in the file content. The js file is already using `[name].[chunkhash].js` but the css is not yet using hash in the file name.

Reply:
Awesome! Feel free to do the CSS hashing as a different work item

Reply:
Let's do this as a different work item! @zuhdil